_layouts/frontpage.html

```diff
@@ -74,7 +74,7 @@ <h1>{{page.headerTitle}}</h1>
             <div class="circle-solid">
               <img src="/resources/img/frontpage/scala-spiral.png" alt="">
               <div class="scala-version">
-                <span>Scala</span>
+                <!-- <span>Scala</span> -->
               </div>
```

Comment:
I guess this should be removed entirely, at this point?

Reply:
done